code/modules/paperwork/paper.dm

```diff
@@ -401,6 +405,52 @@
 /obj/item/weapon/paper/crumpled/update_icon()
 	return
 
+/obj/item/weapon/paper/AltClick(mob/user, obj/item/I,)
+	..()
+	if(!in_range(src, user))
+		return
+	if(!istype(src, /obj/item/weapon/paper/paperplane))
+		user << "<span class='notice'>You fold the paper in the shape of a plane!</span>"
+		if(do_after(user, 20, target = src))
+			user.unEquip(src)
+			get_turf(src)
+			I = new /obj/item/weapon/paper/paperplane(src.loc)
+			user.put_in_hands(I)
+			qdel(src)
+		return
 
 /obj/item/weapon/paper/crumpled/bloody
 	icon_state = "scrap_bloodied"
+
+
+/obj/item/weapon/paper/paperplane
+	name = "\improper paper plane"
+	desc = "paper folded in the shape of a plane"
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "paperplane"
+	throw_range = 7 //paper planes flies pretty far
+	throw_speed = 1 //but not very fast
+	throwforce = 0 // nor does it do any damage
+	gender = NEUTER
+	w_class = 1
+	layer = 3
+	pressure_resistance = 0
+	burn_state = FLAMMABLE
+	burntime = 5
+
+/obj/item/weapon/paper/paperplane/New()
+	..()
+	update_icon()
+
+/obj/item/weapon/paper/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=0)
+	if(!..() || !ishuman(target))//if the plane is caught or it hits a nonhuman
```

Comment:
From the test:
**With if(!..()**
-Target caught paperplane
-Eye Scream Procs occured without trouble

***With if(..())**
-Target Caught Paperplane
-No Eye Scream Procs (35 throws)

I'll deduce the if(!..() was the correct code to put with return, since it returned the best results.